pandas-stubs/core/series.pyi

```diff
@@ -1224,6 +1224,13 @@ class Series(IndexOpsMixin[S1], NDFrame):
         axis: AxisIndex = ...,
         fill_value: object | None = ...,
     ) -> Series: ...
+    def info(
+        self,
+        verbose: bool | None = ...,
+        buf: WriteBuffer[str] = ...,
+        memory_usage: bool | Literal["deep"] | None = ...,
+        show_counts: bool | None = ...,
+    ) -> None: ...
```

Comment:
technically, `Series.info` also has a `max_cols` argument, but it raises if it gets passed, as it's only available for `DataFrame.info`

OK to leave it out of the signature in the types, so as to discourage its usage?

Reply:
first - thanks for all your reviews here!

second - i think it's meant to be documented, it was added very intentionally in https://github.com/pandas-dev/pandas/pull/37320#pullrequestreview-819720418 - i'll make a pr in pandas

Reply:
OK - leaving out `max_cols` sounds like a good idea